### ISSUE TITLE
Update Documentation to include imports & constants in code examples

### ISSUE THREAD

**Issue by joeldom** [react-bootstrap/react-bootstrap#4952]
## Describe the bug

When attemtping to add features, copying code shown, you then have to see if you missed and element because a list isn't shown of what is imported

## To Reproduce

1. copy code example
2. paste in app
3. run
4. error: `Line ?: 'Component' is not defined react/jsx-no-undef`

## Reproducible Example

 [Navbars](https://react-bootstrap.github.io/components/navbar/).

## Expected behavior

I'd like to be able to make a new feature component to quickly import to test it within the context of the app with the imports for the component example shown, as well as any state/functions needed to quickly add features.\

## Documentation Example
[Atlaskit ~ Dropdown Item](https://atlassian.design/components/dropdown-menu/dropdown-item/examples)

## Environment (please complete the following information)

- Operating System: [macOS]
- Browser, Version [FF Dev]
- React-Bootstrap Version ["react-bootstrap": "^1.0.0-beta.16"]

**Comment by nbriannl** [comment]
If I understand correctly, for a given code-block sample, you would like to have all the necessary import statements of all the components used in that code-block copied as well?

**Comment by lslee714** [comment]
Please on this issue, it drives me nuts every time I go in the docs to figure out how to use a component

**Comment by yelkar-snehal** [comment]
Hi, I'm new to open source. Is this issue still open for grabs?

**Comment by VibhorJaiswal** [comment]
I would like to work on it.

**Comment by Aaron-Ennis** [comment]
This issue doesn't look like it's seen any movement for a while. I started picking at it for a school assignment. To make it happen with `react-live` it looks like `<LiveProvider>` in the `ReactPlayground` component needs the `transformCode` prop added and defined as a function that uses some regex to strip the imports and exports. I got it to work using a class component or a function component in the example/live code area, but might have to do some additional transformation to make it work with arrow functions. In any case, is this what we're looking for here? 

![Screen Shot 2022-05-06 at 18 26 00](https://user-images.githubusercontent.com/34149819/167232578-37bf0f95-0cd4-4baf-8de3-caa9da584470.png)
